### PR TITLE
Create ui-c8y-1021-11-0-migrated-custom-properties-form-7484-graft-release-cd-7682.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by pawel-rynarzewski-sag

Ticket: [MTM-60789](https://cumulocity.atlassian.net/browse/MTM-60789)
Original PR: [7682](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7682)

[MTM-60789]: https://cumulocity.atlassian.net/browse/MTM-60789?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1021-11-0-migrated-custom-properties-form-7484-graft-release-cd-7682.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Migrated custom properties form (#7484) [GRAFT][release/cd] (#7682)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1021-11-0-migrated-custom-properties-form-7484-graft-release-cd-7682.md]: The Claude AI suggested to use the following title:
```suggestion
title: Custom properties form migrated to Angular
```

File: content/change-logs/platform-services/ui-c8y-1021-11-0-migrated-custom-properties-form-7484-graft-release-cd-7682.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60789
+version: 1021.11.0
+---
+Migrated custom properties form (#7484) [GRAFT][release/cd] (#7682)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1021-11-0-migrated-custom-properties-form-7484-graft-release-cd-7682.md]: The Claude AI suggested to use the following body:
```suggestion
As part of the ongoing modernization efforts, the custom properties form has been migrated from the legacy technology to Angular. This change brings the custom properties form in line with the latest web technologies used in the application. The migration improves consistency and maintainability of the codebase. For users, the functionality and user interface of the custom properties form remains largely unchanged.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/platform-services/ui-c8y-1021-11-0-migrated-custom-properties-form-7484-graft-release-cd-7682.md (hunk 2 above)
Comment (JanePorter) [inline content/change-logs/platform-services/ui-c8y-1021-11-0-migrated-custom-properties-form-7484-graft-release-cd-7682.md]: The custom properties form has been improved with extraneous information moved to a new 'Limits' tab providing a cleaner and more consistent interface.

**Review by JanePorter — CHANGES_REQUESTED** [review]
See suggested changes

File: content/change-logs/platform-services/ui-c8y-1021-11-0-migrated-custom-properties-form-7484-graft-release-cd-7682.md (hunk 1 above)
Comment (pawel-rynarzewski-c8y) [reply content/change-logs/platform-services/ui-c8y-1021-11-0-migrated-custom-properties-form-7484-graft-release-cd-7682.md]:
```suggestion
title: '"Custom properties" tab improved'
```

File: content/change-logs/platform-services/ui-c8y-1021-11-0-migrated-custom-properties-form-7484-graft-release-cd-7682.md (hunk 2 above)
Comment (pawel-rynarzewski-c8y) [reply content/change-logs/platform-services/ui-c8y-1021-11-0-migrated-custom-properties-form-7484-graft-release-cd-7682.md]: declined this suggestion

Comment (pawel-rynarzewski-c8y) [reply content/change-logs/platform-services/ui-c8y-1021-11-0-migrated-custom-properties-form-7484-graft-release-cd-7682.md]:
```suggestion
The "Custom properties" tab has been improved with extraneous information moved to a new "Limits" tab providing a cleaner and more consistent interface.
```

**Review by JanePorter — APPROVED** [review]

**Review by pawel-rynarzewski-c8y — APPROVED** [review]

File: content/change-logs/platform-services/ui-c8y-1021-11-0-migrated-custom-properties-form-7484-graft-release-cd-7682.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60789
+version: 1021.11.0
+---
+The "Custom properties" tab has been improved with extraneous information moved to a new "Limits" tab providing a cleaner and more consistent interface.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1021-11-0-migrated-custom-properties-form-7484-graft-release-cd-7682.md]:
```suggestion
The **Custom properties** tab in the tenant details has been improved by moving extraneous information to a new **Limits** tab providing a cleaner and more consistent interface.
```

File: content/change-logs/platform-services/ui-c8y-1021-11-0-migrated-custom-properties-form-7484-graft-release-cd-7682.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: '"Custom properties" tab improved'
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1021-11-0-migrated-custom-properties-form-7484-graft-release-cd-7682.md]:
```suggestion
title: Improved Custom properties tab in the tenant details
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**